[Service Bus] [Track 1] Update service-bus with the amqp-common latest (#7823)
Azure/azure-sdk-for-js: sdk/servicebus/service-bus/CHANGELOG.md

# Release History

> ## 1.1.4 (2020-03-16)

Author: ```suggestion
## 1.1.4 (2020-03-17)
```

- `@azure/amqp-common` library has been updated to version 1.0.0-preview.11 with an improvement that increases the default timeout for the requests sent to the service from 10s to 60s to avoid `"$cbs" timeout`.
  This update is to take the changes from the latest `@azure/amqp-common` library to version 1.0.0-preview.11.

> 

Author: ```suggestion
  [PR 7823](https://github.com/Azure/azure-sdk-for-js/issues/7823).
```

>   [PR 7823](https://github.com/Azure/azure-sdk-for-js/issues/7823).

Author: ```suggestion
  [PR 7823](https://github.com/Azure/azure-sdk-for-js/issues/7823).
  
```

# Release History

> ## 1.1.4 (2020-03-13)

Author: ```suggestion
## 1.1.4 (2020-03-16)
```

> - `@azure/amqp-common` library has been updated to version 1.0.0-preview.11 with an improvement that increases the default timeout for the requests sent to the service from 10s to 60s to avoid `"$cbs" timeout`.

Author: ```suggestion
- `@azure/amqp-common` library has been updated to version 1.0.0-preview.11 with an improvement that increases the default timeout for the authorization requests sent to the service from 10s to 60s to avoid frequent timeout errors.
```

>   This release updates `@azure/amqp-common` library to the latest version.

Reviewer: The term "default timeout" here makes one think that they can customize the timeout value which is not the case

```suggestion
- Updated to use the latest version of `@azure/amqp-common` where the timeout for authorization requests sent to the service is increased from 10s to 60s to reduce the frequency of timeout errors.
```

>   This update is to take the changes from the latest `@azure/amqp-common` library to version 1.0.0-preview.11.

Reviewer 2: Suggested wording updates:
```suggestion
- `@azure/amqp-common` has been updated to increase the default timeout for authorization requests sent to the service from 10s to 60s to reduce the frequency of timeout errors.
  This release updates `@azure/amqp-common` library to the latest version.
```

Author: Thank you. 😊
